cmd/gapis/main.go

```diff
@@ -95,13 +97,18 @@ func run(ctx context.Context) error {
 	r := bind.NewRegistry()
 	ctx = bind.PutRegistry(ctx, r)
 	m := replay.New(ctx)
+	t := trace.New(ctx)
 	ctx = replay.PutManager(ctx, m)
 	ctx = database.Put(ctx, database.NewInMemory(ctx))
+	ctx = trace.PutManager(ctx, t)
```

Comment:
Nit: Unless you are going to use `t` in this scope, I'd fold this into `ctx = trace.PutManager(ctx, trace.New(ctx))` to avoid more scope pollution.

Reply:
Done